web/index.html

```diff
@@ -45,27 +107,67 @@
             }
         </style>
     </head>
+
     <body>
-        <header>THIS      IS      A       HEADER</header>
+        <!--Sidebar page links. Still needs styles above.-->
+        <div id="toolBar" class="sidebar">
+            <a href="#">Home</a>
+            <a href="#">About</a>
+            <a href="#">Contribute</a>
+            <a href="#">Contact</a>
+            <a href="#">Tools</a>
+          </div>
+
+        <header>
+        <div class = "header">
+            <div class = "row">
+                <!--This needs to be worked on, as onclick does not appear to remain active for this button without reload.-->
+                <span style="font-size:40px;cursor:pointer;padding:8px 0px 0px 25px" onclick="openCloseMenu()">&#9776;</span>
+                <img src="logo.png" alt="Rerum Playground">
+            </div>
+        </div>
+        </header>
+
         <div class="container">
-            <div id="intro">
+            <div class = "placeholder">
+            </div>
+
+            <!--
+                <div id="intro">
                 Hello.  This is the RERUM playground.  Have fun, be safe, be friendly, follow curfew.  
             </div>
             <div id="tool_set">
                 <h4 title="Tools connected with the playground.  You may be directed to a different site."> Tools </h4>
             </div>
             <div id="interface_set">
                 <h4 title="Interfaces connected with the playground.  You may be directed to a different site."> Interfaces </h4>
-                <a class="catalogEntry"> <div class="thumbLabel">None</div> <img class="thumb" src="https://i0.wp.com/sharetraveler.com/wp-content/uploads/2015/06/sad-face.jpg?ssl=1"> <div class="thumbDescription">We are working on it.  Check back soon!</div> </a>
+                <a class="catalogEntry"> <div class="thumbLabel">None</div> <div class="thumbDescription">We are working on it.  Check back soon!</div> </a>
             </div>
             <div id="technology_set">
                 <h4 title="Technologies connected with the playground.  You may be directed to a different site."> Technologies </h4>
-                <a class="catalogEntry"> <div class="thumbLabel">None</div> <img class="thumb" src="https://i0.wp.com/sharetraveler.com/wp-content/uploads/2015/06/sad-face.jpg?ssl=1"> <div class="thumbDescription">We are working on it.  Check back soon!</div> </a>
+                <a class="catalogEntry"> <div class="thumbLabel">None</div> <div class="thumbDescription">We are working on it.  Check back soon!</div> </a>
             </div>
+            -->
         </div>
+
         <footer>
-            THIS      IS        A          FOOTER
+            <!--THIS      IS        A          FOOTER-->
```

Comment:
since this is just a comment, you could remove it, since the footer element speaks for itself.